pkg/sql/conn_executor_exec.go

```diff
@@ -3418,6 +3419,13 @@ func (ex *connExecutor) execWithProfiling(
 	prepared *PreparedStatement,
 	op func(context.Context) error,
 ) error {
+	if trace.IsEnabled() {
+		defer trace.StartRegion(ctx, "SQLStatement").End()
+		// Go execution trace logging. Low-allocation only, please.
+		trace.Log(ctx, "sqlappname", ex.sessionData().ApplicationName)
```

Comment:
might be better to use regions here: at least they get displayed by `gotraceui`; logs don't as far as I can tell.
I'm not sure how `go tool trace` displays them, haven't checked.